tests/js/HappinessLintBearTest.py

```diff
@@ -17,3 +21,18 @@
 HappinessLintBearTest = verify_local_bear(HappinessLintBear,
                                           valid_files=(good_file,),
                                           invalid_files=(bad_file,))
+
+
+@generate_skip_decorator(HappinessLintBear)
+class HappinessLintBearConfigTest(LocalBearTestHelper):
+
+    VALUE_ERR_RE = ('Set "use_spaces = False" for HappinessLintBear'
+                    'to avoid conflicts with other bears')
```

Comment:
rather inline it^^

Reply:
inlining it makes it 106 char long

Reply:
of course you can wrap it ;)

Reply:
wrap as in "'Set "use_spaces=False" for HappinessLintBear /
                   to avoid conflicts with other bears' ??

Reply:
yeah but don't use backslashes please ;) However we have to choose anyway a different message, so maybe this problems solves itself^^